### ISSUE TITLE
I found this issue to be kind of interesting

### ISSUE THREAD

**Issue by WrViajreo** [AnkitKr0711/Pdf_ChatBot_Query#1]
Hello, I found this repository and the problem it tries to solve to be kinda interesting. Are you interested to find more guys to work on this project together.

**Comment by AnkitKr0711** [comment]
Yes i would love to work on this project more, acquire and develop more
knowledge
